### DISCUSSION TITLE
Download button

### DISCUSSION THREAD

**Discussion by rajarshikhatua** [KRTirtho/spotube#85] [Q&A]
I can't see any download button or player screen on my android (Samsung galaxy m31)

**Comment by KRTirtho** [comment]
Sorry for the late response
Can you share some screenshots of the application & specify which version of Spotube you're using

**Reply by rajarshikhatua** [reply]
https://user-images.githubusercontent.com/77912617/173517662-dbf2a487-3856-493f-90cb-481c4cf9a5f6.mp4

Here is the screen recording

**Comment by KRTirtho** [comment]
To enter the expanded player view mode you've to click/tap on the title of the song or the album art of the mini floating player but you were tapping on the empty space so that is why it wasn't working
